### DISCUSSION TITLE
Brauche Hilfe bei Update Fusion Board V2

### DISCUSSION THREAD

**Discussion by walter2323** [tbnobody/OpenDTU#2219] [Q&A] [answered]
Ich habe heute erfolglos versucht das aktuelle Update einzuspielen. 
Hat jemand Erfolg gehabt?

**Comment by mroenne2022** [comment]
@walter2323 : Mein Fusion Board konnte ich updaten. Welches bin File hast du verwendet? Ich hab dieses (Bild) genommen. 
![Screenshot_20240819_192154_Chrome](https://github.com/user-attachments/assets/d3425b77-0ff2-45ae-8755-f4c0f4423760)

**Comment by walter2323** [comment]
Dankeschön für die Antwort!
Genau diese File wollte ich verwenden, welcher Flasher kam zu Einsatz?

mroenne2022 ***@***.***> schrieb am Mo., 19. Aug. 2024, 19:25:

**Reply by mroenne2022** [reply, marked as answer]
Moment....du schriebst Update: Das geht aus dem Menü über OTA ,OHNE zu flashen !

**Reply by mroenne2022** [reply]
Neu geflasht hab ich mein Fusion Board noch nicht, aber meine anderen DTUs mit den ESP32 hab ich mit diesem Tool geflasht (Bild). Allerdings muss man zum flashen jeweils die zugehörige factory nutzen(im Bild nur ein Bsp File)  Ich weiß aber nicht ob das Tool für die Fusion Boards ebenfalls geht, in der Chip Auswahl ist aber ESP32s3 dabei im Menü.
![Unbenannt](https://github.com/user-attachments/assets/7004fb35-2424-4b6a-9d19-c27b5f256ed2)

**Comment by walter2323** [comment]
Ich bin überwältigt ob der Hilfe!
Update lief jetzt plötzlich, warum auch immer, ohne Probleme OTA durch...

**Reply by mroenne2022** [reply]
Ja so soll es sein , hab nichts anderes erwartet 😀
